sormas-api/src/main/java/de/symeda/sormas/api/document/DocumentFacade.java

```diff
@@ -44,4 +44,6 @@ public interface DocumentFacade {
 	byte[] getContent(String uuid) throws IOException;
 
 	void cleanupDeletedDocuments();
+
+	void deleteDocumentFromAllRelations(String surveyTokenUuid);
```

Comment:
please remove this

Reply:
change the parameter name to documentTokenUuid